db/migrate/20150717135909_remove_unique_index_from_builds.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveUniqueIndexFromBuilds < ActiveRecord::Migration
+  def change
+    # Remove unique index on git_sha, users should be able to rebuild docker images against same SHA
```

Comment:
So that means you could have two different Builds that reference the same git commit, and potentially have different labels. What's the advantage of allowing 2 different Build instances that refer to the exact same version of the code?

And why do users need to be able to rebuild a docker image for that SHA, given that the code is exactly the same?  If that is necessary, couldn't we just create functionality that re-runs the `docker build` command?